src/index.js

```diff
@@ -171,13 +171,20 @@ class ImageCompare extends HTMLElement {
   connectedCallback() {
     this.shadowRoot.appendChild(template.content.cloneNode(true));
     
-    this.shadowRoot.querySelector("input").addEventListener('input', ({ target }) => {
-      if (this.animationFrame) cancelAnimationFrame(this.animationFrame);
-
-      this.animationFrame = requestAnimationFrame(() => {
-        this.shadowRoot.host.style.setProperty('--exposure', `${target.value}%`)
-      });
+    ['input', 'change'].forEach((eventName) => {
+      this.shadowRoot.querySelector("input").addEventListener(
+        eventName,
+        ({ target }) => {
+          if (this.animationFrame) cancelAnimationFrame(this.animationFrame);
+
+          this.animationFrame = requestAnimationFrame(() => {
+            this.shadowRoot.host.style.setProperty('--exposure', `${target.value}%`)
+          });
+        },
+      );
     });
+    this.shadowRoot.querySelector("input").addEventListener('input', handleChange);
+    this.shadowRoot.querySelector("input").addEventListener('change', handleChange);
```

Comment:
@WestonThayer thank you for this PR! 

I didn't realize that iOS VoiceOver didn't fire input events when using the range input this way. That's a great reminder to test using a variety of assistive devices.

I'm a little confused about line 186 and 187:

- Where is `handleChange` being defined?
- Are these necessary in addition to the event listeners added in the loop above?

Thanks!

Reply:
Oh no! Those aren't supposed to be there. Remnants from before I switched to using a `['input', 'change']` array. Removing now...

Reply:
Ok gone 😅, thanks for catching it.

Yes, it is quirky. I suspect it's a quirk with Safari/WebKit. I'd file a bug there, but I don't think there's a spec to cover what events an input should fire when incremented/decremented via the OS accessibility APIs... issues like this add weight to the argument that there should be though.

Reply:
Thanks! I'll test this out later today or tomorrow and get this merged in 🙂

Reply:
Sounds good! Note that at least on iOS 15.0.2, Safari wasn't firing *any* events when using VoiceOver with the range input, so check your version if it's not working.

Reply:
>Note that at least on iOS 15.0.2, Safari wasn't firing any events when using VoiceOver with the range input

Oh wow! VoiceOver wasn't firing `change` or `input`? That definitely seems buggy!

Reply:
Agreed, I was glad to find it at least partially fixed in iOS 15.1.

I opened https://bugs.webkit.org/show_bug.cgi?id=233449